internal

```diff
@@ -0,0 +1 @@
+/usr/local/google/home/aee/cobalt/cobalt_src/internal
```

Comment:
I didn't review the rest but it looks like there's something wrong here with "internal" sneaking in.